README.md

```diff
@@ -1,69 +1,104 @@
+[vmware.vcloud_director.vcd_org](https://github.com/vmware/ansible-module-vcloud-director/blob/main/docs/vmware.vcloud_director.vcd_org_module.rst)|Manage org's states/operations in vCloud Director
+[vmware.vcloud_director.vcd_org_vdc](https://github.com/vmware/ansible-module-vcloud-director/blob/main/docs/vmware.vcloud_director.vcd_org_vdc_module.rst)|Manage ORG VDC's states/operations in vCloud Director
+[vmware.vcloud_director.vcd_resources](https://github.com/vmware/ansible-module-vcloud-director/blob/main/docs/vmware.vcloud_director.vcd_resources_module.rst)|Add/Delete/Update VCD Infrastructure resources
+[vmware.vcloud_director.vcd_roles](https://github.com/vmware/ansible-module-vcloud-director/blob/main/docs/vmware.vcloud_director.vcd_roles_module.rst)|Manage role's states/operations in vCloud Director
+[vmware.vcloud_director.vcd_user](https://github.com/vmware/ansible-module-vcloud-director/blob/main/docs/vmware.vcloud_director.vcd_user_module.rst)|Manage user's states/operations in vCloud Director
+[vmware.vcloud_director.vcd_vapp](https://github.com/vmware/ansible-module-vcloud-director/blob/main/docs/vmware.vcloud_director.vcd_vapp_module.rst)|Manage vApp's states/operations in vCloud Director
+[vmware.vcloud_director.vcd_vapp_network](https://github.com/vmware/ansible-module-vcloud-director/blob/main/docs/vmware.vcloud_director.vcd_vapp_network_module.rst)|Manage vApp Network's states/operation in vCloud Director
+[vmware.vcloud_director.vcd_vapp_vm](https://github.com/vmware/ansible-module-vcloud-director/blob/main/docs/vmware.vcloud_director.vcd_vapp_vm_module.rst)|Manage VM snapshots states/operations in vCloud Director
+[vmware.vcloud_director.vcd_vapp_vm_disk](https://github.com/vmware/ansible-module-vcloud-director/blob/main/docs/vmware.vcloud_director.vcd_vapp_vm_disk_module.rst)|Ansible Module to manage disks in vApp VMs in vCloud Director.
+[vmware.vcloud_director.vcd_vapp_vm_nic](https://github.com/vmware/ansible-module-vcloud-director/blob/main/docs/vmware.vcloud_director.vcd_vapp_vm_nic_module.rst)|Manage VM NIC's states/operations in vCloud Director
+[vmware.vcloud_director.vcd_vdc_gateway](https://github.com/vmware/ansible-module-vcloud-director/blob/main/docs/vmware.vcloud_director.vcd_vdc_gateway_module.rst)|Manage edge gateway's states/operations in vCloud Director
+[vmware.vcloud_director.vcd_vdc_network](https://github.com/vmware/ansible-module-vcloud-director/blob/main/docs/vmware.vcloud_director.vcd_vdc_network_module.rst)|Manage VDC Network's states/operations in vCloud Director
 
-1. vcd_catalog
-2. vcd_catalog_item
-3. vcd_disk
-4. vcd_external_network
-5. vcd_org
-6. vcd_org_vdc
-7. vcd_roles
-8. vcd_user
-9. vcd_vapp
-10. vcd_vapp_network
-11. vcd_vapp_vm
-12. vcd_vapp_vm_disk
-13. vcd_vapp_vm_nic
-14. vcd_vapp_vm_snapshot
-15. vcd_vdc_gateway
-16. vcd_vdc_network
-17. vcd_gateway_services
+<!--end collection content-->
+
+## Testing and Development
```

Comment:
We don't have this testing setup for vcd yet

Reply:
OK, I will add TBD section instead.